src/main/java/wooteco/subway/line/domain/Line.java

```diff
@@ -0,0 +1,95 @@
+package wooteco.subway.line.domain;
+
+import wooteco.subway.exception.IllegalLineArgumentException;
+import wooteco.subway.line.controller.dto.LineRequest;
+import wooteco.subway.station.domain.Station;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Line {
+    private Long id;
+    private String name;
+    private String color;
+    private List<Station> stations;
+
+    public Line() {
+    }
+
+    public Line(String name, String color) {
+        this(0L, name, color, new ArrayList<>());
+    }
+
+    public Line(Long id, String name, String color, List<Station> stations) {
+        validatedNameLength(name);
+        validatedColorLength(color);
+        this.id = id;
+        this.name = name;
+        this.color = color;
+        this.stations = stations;
+    }
+
+    public static Line from(LineRequest lineRequest) {
+        validatedNameLength(lineRequest.getName());
+        validatedColorLength(lineRequest.getColor());
+        return new Line(lineRequest.getName(), lineRequest.getColor());
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public List<Station> getStations() {
+        return stations;
+    }
+
+    public boolean equalName(String name) {
+        return this.name.equals(name);
+    }
+
+    public boolean equalId(Long id) {
+        return this.id.equals(id);
+    }
+
+    private static void validatedNameLength(String name) {
+        if(name == null || name.length() == 0) {
+            throw new IllegalLineArgumentException("유효한 이름이 아닙니다.");
```

Comment:
```suggestion
    private static void validatedNameLength(String name) {
        if (name == null || name.length() == 0) {
            throw new IllegalLineArgumentException("유효한 이름이 아닙니다.");
```

예외를 던지고는 있는데 이를 처리해 주지 않고 있네요.

Reply:
앗.. 까먹었네요!

Reply:
테스트 코드를 잘 작성했다면 잊어버릴 일이 없지 않았을까요? 😉